src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,35 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.dao.ManufacturerDaoImpl;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.models.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerDaoImpl manufacturerDao
```

Comment:
You need to use Interface instead of Implementation for loose coupling
```suggestion
    private final ManufacturerDao manufacturerDao
```

Reply:
agree